packages/core/src/index.tsx

```diff
@@ -1,98 +1,111 @@
     const repo =
       process.env.REPO ||
       (`${process.env.VERCEL_GIT_REPO_OWNER}/${process.env.VERCEL_GIT_REPO_SLUG}` as string);
 
-    const [ghUserName, repoSlug] = repo.split("/")
+    const [ghUserName, repoSlug] = repo.split("/");
 
     return {
       ghToken,
       repo,
       ghUserName,
-      repoSlug
+      repoSlug,
     };
   })();
 
-  private PAGE_PATH_PLACEHOLDER = 'path'
+  private PAGE_PATH_PLACEHOLDER = "path";
 
-  private REVALIDATE = 60
+  private REVALIDATE = 60;
 
-  private allowUsers: string[] = [this.resolvedConfig.ghUserName].concat(this.config?.allowUsers || [])
+  private allowUsers: string[] = [this.resolvedConfig.ghUserName].concat(
+    this.config?.allowUsers || []
+  );
 
   private authHeaders = this.resolvedConfig.ghToken
     ? {
-      Authorization: `bearer ${this.resolvedConfig.ghToken}`,
-    }
+        Authorization: `bearer ${this.resolvedConfig.ghToken}`,
+      }
     : ({} as any);
 
-  private async request<T>(query: {
-    query: string,
-    variables: any
-  }) {
-    const res = await axios.post('https://api.github.com/graphql', query, {
-      headers: this.authHeaders
-    })
-    return res.data.data as T
+  private async request<T>(query: { query: string; variables: any }) {
+    const res = await axios.post("https://api.github.com/graphql", query, {
+      headers: this.authHeaders,
+    });
+    return res.data.data as T;
   }
 
-  constructor(public config: SairinConfig) {
-  }
+  constructor(public config: SairinConfig) {}
 
   getPostList = async () => {
+    const result = await this.request<GetPostQueryProps>(
+      GetPostsQuery({
+        owner: this.resolvedConfig.ghUserName,
+        repo: this.resolvedConfig.repoSlug,
+      })
+    );
 
-    const result = await this.request<GetPostQueryProps>(GetPostsQuery({
-      owner: this.resolvedConfig.ghUserName,
-      repo: this.resolvedConfig.repoSlug,
-    }))
-
-    const posts = result.repository.issues.nodes.filter(post => {
-      return this.allowUsers.indexOf(post.author.login) !== -1
-    }).map((post) => {
-      const { html, attributes } = this.processBody(post.body);
-      post.comments.nodes = post.comments.nodes.map(comment => {
-        const { html } = this.processBody(comment.body)
-        return {
-          ...comment,
-          html
-        }
+    const posts = result.repository.issues.nodes
+      .filter((post) => {
+        return this.allowUsers.indexOf(post.author.login) !== -1;
       })
+      .map((post) => {
+        const { html, attributes } = this.processBody(post.body);
+        post.comments.nodes = post.comments.nodes.map((comment) => {
+          const { html } = this.processBody(comment.body);
+          return {
+            ...comment,
+            html,
+          };
+        });
 
-      return {
-        ...post,
-        html,
-        attributes,
-      };
-    });
+        return {
+          ...post,
+          labels: post.labels.nodes.reduce(
```

Comment:
I think we can just use `.filter`:

```js
post.labels.nodes.filter(label => label !== 'published').map(label => ({ ...label, `#${label.color}` }))
```
Another suggestion is we could remove the `map` here. It is too expensive for only adding a `#` symbol. We could add this `#` when we use (e.g. in the template).

Reply:
sure! just in case you have other comments plus its late now in China, I'll address the comment tmr, have go good night~